apps/music/test/unit/mock_lazy_loader.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+/* exported MockLazyLoader */
+
+// Our MockLazyLoader is better than the shared one because we actually use a
+// Promise for load(). However, trying to do that with the shared version breaks
+// all the system app's tests. Oh well.
+var MockLazyLoader = {
+  load: function(fileArray, callback) {
+    if (callback) {
+      callback();
+    } else {
+      return Promise.resolve();
+    }
+  },
+
+  getJSON: function(file) {
+    return Promise.resolve({});
+  }
+};
```

Comment:
@jimporter Thanks for filing it! I agree instead of breaking dozen of the system app tests, a real Promise version MockLazyLoader for music only would be enough, the system guys should fix that bug.